src/ApiConnectors/MatchesApiConnector.php

```diff
@@ -16,18 +19,20 @@ class MatchesApiConnector extends BaseApiConnector
 {
     /**
      * @param MatchSet $matchSet
+     *
      * @throws Exception
      */
-    public function send(MatchSet $matchSet): void
+    public function send(MatchSet $matchSet): MatchSet
     {
-        $this->sendAll([$matchSet]);
+        return $this->sendAll([$matchSet])[0]->unwrap();
     }
 
     /**
-     * @param MatchSet[] $matchSets
+     * @return MappedResponseCollection|IndividualMappedResponse[]
```

Comment:
Hmm, I suppose `IndividualMappedResponse[]` is not really necessary?

Reply:
It helps when doing `foreach ($mappedResponseCollection as $item)`. Otherwise Php(Storm) won't know that MappedResponseCollection iterates over.

Reply:
It's really useful for hinting as PhpStorm will understand this when iterating of the return value.